exporter/importables.go

```diff
@@ -1133,6 +1133,17 @@ var resourcesMap map[string]importable = map[string]importable{
 			name := r.ID[1:] + ext[language] // todo: replace non-alphanum+/ with _
 			content, _ := base64.StdEncoding.DecodeString(contentB64)
 			fileName, err := ic.createFileIn("notebooks", name, []byte(content))
+			splits := strings.Split(r.Name, "_")
+			notebookId := splits[len(splits)-1]
+
+			if ic.meAdmin {
+				ic.Emit(&resource{
+					Resource: "databricks_permissions",
+					ID:       fmt.Sprintf("/notebooks/%s", notebookId),
+					Name:     "notebook_" + ic.Importables["databricks_notebook"].Name(ic, r.Data),
+				})
```

Comment:
Main problem here is that it will generate too many Terraform resources that aren't necessary, and this may lead to very slow execution. Otherwise it would be implemented long timeago

Reply:
got it, do we have solutions for this?

Reply:
my 2 cents, perhaps we can have an option to export permissions and this wouldn't be on by default?

Reply:
Not necessary to have a separate option, just need to do incremental export

Reply:
actually, for notebook permission, only permissions granted to users other than the owner/author of the notebook are exported (might need your advice on how this works! )